### PR TITLE
Update alert-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Find and release quarantined messages as a user in EOP and the triggered alerts are applicable to EOP. Looks like the Basic, Standard and Premium also have this feature.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9274

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/alert-policies.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ The table also indicates the Office 365 Enterprise and Office 365 US Government
...
+|**User requested to release a quarantined message**|Generates an alert when a user requests release for a quarantined message. To request the release of quarantined messages, the **Allow recipients to request a message to be released from quarantine** (_PermissionToRequestRelease_) permission is required in the quarantine policy (for example, from the **Limited access** preset permissions group). For more information, see [Allow recipients to request a message to be released from quarantine permission](../security/office-365-security/quarantine-policies.md#allow-recipients-to-request-a-message-to-be-released-from-quarantine-permission). This policy has an **Informational** severity setting.|Threat management|No| Microsoft Busniness Basic, Microsoft BusninesStandard, Microsoft Busnines Premium, E1/F1/G1, E3/F3/G3, or E5/G5|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/alert-policies.md]:
```suggestion
|**User requested to release a quarantined message**|Generates an alert when a user requests release for a quarantined message. To request the release of quarantined messages, the **Allow recipients to request a message to be released from quarantine** (_PermissionToRequestRelease_) permission is required in the quarantine policy (for example, from the **Limited access** preset permissions group). For more information, see [Allow recipients to request a message to be released from quarantine permission](../security/office-365-security/quarantine-policies.md#allow-recipients-to-request-a-message-to-be-released-from-quarantine-permission). This policy has an **Informational** severity setting.|Threat management|No| Microsoft Business Basic, Microsoft Business Standard, Microsoft Business Premium, E1/F1/G1, E3/F3/G3, or E5/G5|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Pull request merged**